### PR TITLE
Fix error with biopython 1.81

### PR DESCRIPTION
**Pull request by abretaud**
A possible solution for #136, tested on a minimal sample with python 3.9. Any review/test is welcome to make sure it works in all situations.

With this patch, bcbio-gff should work equally with biopython <=1.80 and biopython 1.81

I know bcbio-gff is not actively maintained anymore, but there's still a ton of code using it, so having a new release with this fix would make life easier

### CODE REVIEW

File: gff/BCBio/GFF/GFFParser.py [hunk 1]
```diff
@@ -69,7 +77,7 @@ def _split_keyvals(keyval_str):
 
         GFF3 has key value pairs like:
           count=9;gene=amx-2;sequence=SAGE:aacggagccg
-        GFF2 and GTF have:
```
Comment (corneliusroemer) [inline gff/BCBio/GFF/GFFParser.py:72]: If you don't use auto-formatting your PR would be easier to remove! Thanks for giving this a stab!

Comment (tsibley) [reply gff/BCBio/GFF/GFFParser.py:72]: (s/remove/review/ presumably)

Comment (abretaud) [reply gff/BCBio/GFF/GFFParser.py:72]: yeah, sorry, I figured auto-formatting is enabled everywhere by default now anyway, and the diff is not so huge

**Comment by abretaud** [comment]
ping @chapmanb in case you were not notified :)

**Comment by abretaud** [comment]
Wonderful thanks a lot for the merge and release @chapmanb! :+1:

**Pull request merged**